core/templates/dev/head/exploration_editor/main_tab/state_editor.html

```diff
@@ -8,7 +8,7 @@
   </div>
 
   <md-card class="oppia-editor-card-with-avatar">
-    <div class="oppia-editor-card-body" ng-show="!!stateName">
```

Comment:
Is ``stateName`` guaranteed to be defined? I'm wondering why this was here and why it's being removed.